main.tf

```diff
@@ -0,0 +1,110 @@
+resource "azurerm_storage_account" "storaccbackup" {
+  account_replication_type = "LRS"
+  account_tier             = "Standard"
+  location                 = var.location
+  name                     = "${lower(var.project)}${lower(var.stage)}storaccbackup"
+  resource_group_name      = var.resource_group
+}
+
+resource "azurerm_storage_container" "storcontbackup" {
+  name                 = "${lower(var.project)}${lower(var.stage)}storcontbackup"
+  storage_account_name = azurerm_storage_account.storaccbackup.name
+}
+
+resource "kubernetes_namespace" "velero" {
+  metadata {
+    name = "velero"
+  }
+}
+
+resource "helm_release" "velero" {
+  name       = "velero"
+  repository = "https://vmware-tanzu.github.io/helm-charts"
+  chart      = "velero"
+  version    = var.velero_version
+  namespace  = kubernetes_namespace.velero.metadata[0].name
+
+  set {
+    name  = "configuration.provider"
+    value = "azure"
+  }
+  set {
+    name  = "configuration.backupStorageLocation.name"
+    value = "default"
+  }
+  set {
+    name  = "configuration.backupStorageLocation.bucket"
+    value = azurerm_storage_container.storcontbackup.name
+  }
+  set {
+    name  = "configuration.backupStorageLocation.prefix"
+    value = "kubernetes"
+  }
+  set {
+    name  = "configuration.backupStorageLocation.config.resourceGroup"
+    value = var.resource_group
+  }
+  set {
+    name  = "configuration.backupStorageLocation.config.storageAccount"
+    value = azurerm_storage_account.storaccbackup.name
+  }
+  set {
+    name  = "configuration.backupStorageLocation.config.subscriptionId"
+    value = var.subscription_id
+  }
+  set {
+    name  = "snapshotsEnabled"
+    value = var.snapshots_enabled
+  }
+  set {
+    name  = "credentials.secretContents.cloud"
+    value = <<EOF
+AZURE_SUBSCRIPTION_ID=${var.subscription_id}
+AZURE_TENANT_ID=${var.backup_tenant_id}
+AZURE_CLIENT_ID=${var.backup_application_id}
+AZURE_CLIENT_SECRET=${var.backup_application_secret}
+AZURE_RESOURCE_GROUP=${var.resource_group}
+AZURE_CLOUD_NAME="AzurePublicCloud"
+EOF
+  }
+  set {
+    name  = "initContainers[0].name"
+    value = "velero-plugin-for-azure"
+  }
+  set {
+    name  = "initContainers[0].image"
+    value = "velero/velero-plugin-for-microsoft-azure:${var.velero_plugin_version}"
+  }
+  set {
+    name  = "initContainers[0].volumeMounts[0].mountPath"
+    value = "/target"
+  }
+  set {
+    name  = "initContainers[0].volumeMounts[0].name"
+    value = "plugins"
+  }
+  set {
+    name  = "schedules.backup.schedule"
+    value = var.schedule
+  }
+  dynamic "set" {
+    for_each = var.included_namespaces
+    content {
+      name  = "schedules.backup.template.includedNamespaces[${set.key}]"
+      value = set.value
+    }
+  }
+
```

Comment:
Cosmetics: Remove blank line to keep it unified

Reply:
Did we remove all the blank lines between the resources in the other modules? I find this much more readable actually. 🤔

Reply:
Well, i don't know, but i vote for having it consistent: Either all the "sets" get a blank line in between and those two dynamic ones or none of them...

Reply:
Ahhhh! That was what you meant. Sorry, didn't get that. I'll fix it.